java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md

```diff
@@ -237,4 +237,82 @@ bin/start.sh \
 --templateProperty kafka.pubsub.output.projectId=$GCP_PROJECT \
 --templateProperty kafka.pubsub.starting.offset=earliest \
 --templateProperty kafka.pubsub.await.termination.timeout=120000
+```
+
+## 4. Kafka To BQ via spark Direct stream
+
+General Execution:
+
+```
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+SUBNET=<subnet>   \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+HISTORY_SERVER_CLUSTER=<history-server> \
+bin/start.sh \
+-- --template KafkaToBQDstream \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty kafka.bootstrap.servers=<kafka broker list> \
+--templateProperty kafka.topic=<kafka topic name> \
+--templateProperty kafka.starting.offset=<latest | earliest> \
+--templateProperty kafka.bq.stream.output.mode=<Append | Overwrite | ErrorIfExists | Ignore> \
+--templateProperty kafka.gcs.batch.interval=<Batch interval of the stream> \
+--templateProperty kafka.gcs.consumer.group.id=<Consumer group id for the kafka topic> \
+--templateProperty kafka.bq.dataset=kafkatobq \
+--templateProperty kafka.bq.table=kafkaevents \
+--templateProperty kafka.bq.temp.gcs.bucket=<gcs-bucket-name> \
+--templateProperty kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
+```
+
+### Configurable Parameters
+Following properties are avaialble in commandline or [template.properties](../../../../../../../resources/template.properties) file:
+
+```
+# Kafka to BigQuery via Dstream
+
+# Kafka servers
+kafka.bootstrap.servers=<kafka broker list>
+
+# Kafka topics
+kafka.topic=<kafka topic names>
+
+# BigQuery output dataset
+kafka.bq.dataset=<output bigquery dataset>
+
+# BigQuery output table
+kafka.bq.table=<output bigquery table>
+
+# Cloud Storage bucket name, for storing temporary files
+kafka.bq.temp.gcs.bucket=<cloud storage bucket name>
+
+# Offset to start reading from. Accepted values: "earliest", "latest" 
+kafka.bq.starting.offset=<kafka-starting-offset>
+
+# Waits for specified time in ms before termination of stream
+kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
+
+# Ouptut mode for writing data. Accepted values: 'overwrite', 'append', 'ignore', 'error', 'errorifexists', 'default'
+kafka.bq.stream.output.mode=<output-mode> default value : append
+```
+
```

Comment:
Add description for newly introduced template properties like: `kafka.gcs.batch.interval` `kafka.gcs.consumer.group.id`

Reply:
done